src/controllers/mail.controller.ts

```diff
@@ -56,9 +47,55 @@ export const getEmailsFromGmailHandler = async (
 			size: response.data.messages.length,
 			nextPageToken: response.data.nextPageToken,
 		});
-	} catch (err) {
-		logger.error(err);
+	} catch (err: any) {
+		logger.error(err.response);
+
+		return res.status(StatusCodes.INTERNAL_SERVER_ERROR).json({
+			error: "Internal Server Error",
+		});
+	}
+};
+
+/**
+ * This controller will send a email from users gmail account
+ * @param req express request
+ * @param res express response
+ *
+ * @author aayushchugh
+ */
+export const postSendGmailHandler = async (
+	req: Request<PostSendGmailSchema["params"], {}, PostSendGmailSchema["body"]>,
+	res: Response,
+) => {
+	const { to, subject, html } = req.body;
```

Comment:
Hey! Please use a santization library to sanitize the html recieved on the backend just as a safety precaution before sending it to the client. You can use the following libraries, [DOMpurify](https://www.npmjs.com/package/dompurify) & [JSdom](https://www.npmjs.com/package/jsdom). You can either use those two libraries, or take a look at this. [Isomorphic-DOMpurify](https://github.com/kkomelin/isomorphic-dompurify) which is esentially a both the previous libraries combined into one. They both do the same thing. Good Luck!

Reply:
Ok, will do this

Reply:
@is-it-ayush added sanitization please check now